test/errors/scope-resolution/redefinition/redefinition-variables.chpl

```diff
@@ -0,0 +1,2 @@
+--detailed-errors # redef-detailed.good
```

Comment:
This looks like a stray from the old compopts, and its corresponding `good` files look unintentional